android-components/components/service/nimbus/src/main/java/mozilla/components/service/nimbus/messaging/NimbusMessagingStorage.kt

```diff
@@ -108,7 +108,7 @@ class NimbusMessagingStorage(
         )
     }
 
-    @VisibleForTesting(otherwise = PRIVATE)
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
```

Comment:
Was there a reason for this change? Otherwise, not sure if this should change as part of this task